test/check/components.go

```diff
@@ -76,13 +80,20 @@ var (
 			"macvtap-cni",
 		},
 	}
+	MonitoringComponent = Component{
+		ComponentName:  "Monitoring",
+		Service:        "cluster-network-addons-operator-prometheus-metrics",
+		ServiceMonitor: "prometheus-rules-cluster-network-addons-operator",
```

Comment:
those two lines should be same value?
ServiceMonitor, PrometheusRule

Reply:
I mainly did it cause kubevirt's [serviceMonitor](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-operator/resource/generate/components/crds.go#L496) and [prometheusRule](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-operator/resource/generate/components/crds.go#L539) have the same name, but It's not really mandatory and I haven't seen it done in other projects...
so.. do you think I should change it?

Reply:
maybe call them `prometheus-cnao-service-monitor`
and
`prometheus-cnao-rule` ?

Reply:
I think we should keep it consistent, we don't use cnao abbrev on CNAO instances, 
so I think we should only change the `ServiceMonitor` to `service-monitor-cluster-network-addons-operator`